pkg/security/admission/admission_test.go

```diff
@@ -141,16 +141,17 @@ func TestAdmitCaps(t *testing.T) {
 	for i := 0; i < 2; i++ {
 		for k, v := range tc {
 			v.pod.Spec.Containers, v.pod.Spec.InitContainers = v.pod.Spec.InitContainers, v.pod.Spec.Containers
+
+			testSCCAdmit(k, v.sccs, v.pod, v.shouldPass, t)
```

Comment:
Because `testSCCAdmit()` modifies containers, I had to move it above, so when we're assigning containers to a local variable we get the updated value.

As far I understand, prior this change the admissions plugin was modifying a members of the passed argument and now it completely replacing an argument.

Reply:
yeah, that's fine... shouldn't start saving pointers to internal bits of the admitted object until after it is admitted